cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/flow/ConsumerSupplier.java

```diff
@@ -45,31 +46,31 @@
   private final DataFabricFacade dataFabricFacade;
   private final QueueName queueName;
   private final int numGroups;
-  private final UsageRegistry usageRegistry;
-  private final Id.Namespace namespace;
+  private final RuntimeUsageRegistry runtimeUsageRegistry;
   private final List<Id> owners;
   private ConsumerConfig consumerConfig;
   private Closeable consumer;
 
-  static <T> ConsumerSupplier<T> create(Id.Namespace namespace, List<Id> owners, UsageRegistry usageRegistry,
+  static <T> ConsumerSupplier<T> create(List<Id> owners,
```

Comment:
It wasn't being used from the class when I started this change, so I simply removed the parameter passed in.
Intelij complained it was unused, I think.